multiqc/multiqc.py

```diff
@@ -746,6 +746,27 @@ def __rich_measure__(self, console: rich.console.Console, options: rich.console.
         report.runtimes["mods"][run_module_names[mod_idx]] = time.time() - mod_starttime
     report.runtimes["total_mods"] = time.time() - total_mods_starttime
 
+    # Parse software version from config if provided
```

Comment:
I would prefer to move this whole block of code into a function in `software_versions.py` and just pass the required `report` objects to it.

I'm hoping to refactor `mutliqc.py` to split it up into more files so that it's easier to navigate.

'cc @vladsavelyev to check he agrees here. This might all need a bit of reorganising anyway when we refactor to use proper `report` classes in the future.

Reply:
Moved block in c70c0f30c4a7da9cf0b0801e08ff80ddd97fcd7c